genestack_client/data_importer.py

```diff
@@ -124,6 +125,40 @@ def __invoke_loader(self, parent, importer_type, metainfo):
         self.__process_links(metainfo)
         return self.importer.invoke('createFile', parent, importer_type, metainfo)['accession']
 
+    @staticmethod
+    def __add_to_metainfo(metainfo, key, value, value_type, required=False):
+        """
+        Add value to metainfo, check if value is not already present,
+        checks required fields.
```

Comment:
Add ``key``: ``value`` pair to ``metainfo``.

Fails if:
* ``metainfo`` already contains ``key``;
* ``key`` is ``required``, but no value is provided by user,
  and neither ``metainfo`` has already existing ``key`` (with some value).